### PR TITLE
Stabilization of partial proprietary policy support on master

### PR DESCRIPTION
**Pull request by dev-gh**
Fixed issues found during 1st test iteration.

### CODE REVIEW

**Comment by dev-gh** [comment]
@AGaliuzov 
Please review, but **do not merge** until testing will be finished

File: src/components/policy/src/policy/src/policy_manager_impl.cc [hunk 1]
```diff
@@ -705,15 +713,25 @@ std::string PolicyManagerImpl::GetPolicyTableStatus() const {
...
+  uint32_t next = 0u;
+  if (retry_sequence_seconds_.empty() ||
+      retry_sequence_index_ >= retry_sequence_seconds_.size()) {
+    LOG4CXX_WARN(logger_, "Next retry timeout is " << next);
+    return next;
   }
+
+  ++retry_sequence_index_;
+
+  for (uint32_t i = 0u; i < retry_sequence_index_; ++i) {
```
Comment (AGaliuzov) [inline src/components/policy/src/policy/src/policy_manager_impl.cc]: `next = std::accumulate(retry_sequence.begin(), retry_sequence.begin() + retry_sequence_index, next)`

**Comment by AGaliuzov** [comment]
Reviewed

**Comment by dev-gh** [comment]
@anosach-luxoft 
Please review

**Comment by dev-gh** [comment]
@VProdanov @vlantonov @OHerasym 
Please review

**Comment by anosach-luxoft** [comment]
Reviewed

**Comment by VProdanov** [comment]
Reviewed

**Comment by OHerasym** [comment]
Reviewed

**Comment by vlantonov** [comment]
Reviewed

**Pull request merged**